src/components/molecules/GitFormItems/tooltipUtils.tsx

```diff
@@ -0,0 +1,24 @@
+import {CheckCircleFilled, LoadingOutlined, WarningFilled} from '@ant-design/icons';
+
+import Colors from '@styles/Colors';
+
+export type tooltipStatus = 'loading' | 'error' | 'success' | 'none';
```

Comment:
Better to use `enum`:
```suggestion
enum TooltipStatus {
  loading = 'loading',
  error = 'error',
  success = 'success',
  none = 'none',
}
export type tooltipStatus = 'loading' | 'error' | 'success' | 'none';
```